hydra-node/src/Hydra/Chain.hs

```diff
@@ -32,13 +32,13 @@ import Hydra.Ledger (ChainSlot, IsTx, UTxOType)
 import Hydra.OnChainId (OnChainId)
 import Hydra.Party (Party)
 import Hydra.Snapshot (ConfirmedSnapshot, SnapshotNumber)
-import Test.QuickCheck (scale, suchThat)
+import Test.Cardano.Ledger.Core.Arbitrary ()
```

Comment:
Could: `import Test.Cardano.Ledger.Babbage.Arbitrary ()` instead

This would avoid the additional dependency on the `cardano-ledger-core:testlib` in exchange of mentioning `Babbage` here :shrug: